bat/BLDJAR.BAT

```diff
@@ -20,8 +21,11 @@ if %ERRORLEVEL% NEQ 0 (
     echo BLDJAR   ERROR: Error compiling z390
     exit /b %ERRORLEVEL%
 )
+rem create z390.properties file
+CALL ..\BAT\RELVER.BAT %1
```

Comment:
See comments at bash/relver.

In short, the z390.properties file should exist before BLDJAR.BAT is run.